### ISSUE TITLE
Blurry Text/Fonts - Brave Windows

### ISSUE THREAD

**Issue by giwrgosmant** [brave/brave-browser#27950]
## Description 

I've made a detailed post in the [brave community forum](https://community.brave.com/t/blurry-text-fonts-brave-windows/460711).
Basically the text-fonts and generally are seem blurry in the Brave compared to other browsers especially to Firefox which is crystal clear.

## Steps to Reproduce

   1. Open Brave and go for example on this [xda forum post](https://forum.xda-developers.com/t/windows-ultimate-collection-guides.4507867/](https://forum.xda-developers.com/t/revanced-revanced-extended-guides-useful-links.4523967/))
   2. See the text and compare it to Firefox

## Actual result:

Are blurry.
Here some [screenshots ](https://mega.nz/folder/XgszURqZ#0DdvYtdBMVKEw3z3uwW4uA)
Brave:
![1-BRAVE](https://user-images.githubusercontent.com/51835189/213884947-1fd3c73d-b128-4444-93e2-d5eae7942cac.png)
Firefox:
![1-FIREFOX](https://user-images.githubusercontent.com/51835189/213884955-52504fdb-e80b-4189-bffb-c5fbd11a426e.png)

## Expected result:
It should be clear like in Firefox.

## Reproduces how often: 

Always.

## Brave version (brave://version info)

 v1.47.171 (Jan 12, 2023) also tried on Beta and Nightly.

## Version/Channel Information:

- Can you reproduce this issue with the current release?  Yes
- Can you reproduce this issue with the beta channel?  Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No so much

## Miscellaneous Information:

Basically as I said check the [brave community forum](https://community.brave.com/t/blurry-text-fonts-brave-windows/460711/15) which we have conversation with others. Also you can try to google "blurry text" on Brave at Google search you will see it's common issue.
Check screen record video and screenshots [here](https://mega.nz/folder/m1UGSayK#7mZxlR7k5_8Q4syVBXK8RQ).

**Comment by Brave-Matt** [comment]
cc @pilgrim-brave to take a look here

**Comment by xBounceIT** [comment]
Up! This issue seems to appear on every single Chromium browser on my end.

Been struggling with this issue for a while as im force to use Firefox.

**Comment by giwrgosmant** [comment]
> Up! This issue seems to appear on every single Chromium browser on my end.
> 
> Been struggling with this issue for a while as im force to use Firefox.

Yeap, it seems it is the Chromium rendering engine.
I don't know how much this can be improved.

**Comment by Solid-Snake-99** [comment]
THIS IS A GPU DRIVER/SOFTWARE PROBLEM (AFFECTS AMD RADEON RX 6000 USERS)
PERMANENT SOLUTION:

Install STYLUS extension, add www.youtube.com or All sites to execute this code:

video {
  backdrop-filter: blur(0px) !important;
}

ENJOY CLEAR VIDEOS AND PAGES THAT USE GPU HARDWARE ACCELLERATION
